polaris_tokens.gemspec

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |s|
+  s.name = 'polaris_tokens'
+  s.platform = Gem::Platform::RUBY
+  s.version = '0.17.0'
```

Comment:
Can the version please come from package.json?

See https://github.com/Shopify/polaris-rails/pull/1291/files#diff-f06e7790a302d52a6cc1287dd80c556d for how it's done in polaris-rails